docs/current_docs/api/arguments.mdx

```diff
@@ -432,15 +571,43 @@ Here's an example of a Dagger Function that accepts a `File` as argument, reads
 
 Here is an example of passing a local file to this Dagger Function as argument:
 
+<Tabs groupId="shell">
+<TabItem value="System shell">
+```shell
+dagger -c 'read-file /my/file/path/README.md'
+```
+</TabItem>
+<TabItem value="Dagger Shell">
+```shell title="Type 'dagger' to start Dagger Shell in interactive mode and enter the command below."
+read-file /my/file/path/README.md
+```
+</TabItem>
+<TabItem value="Dagger CLI">
 ```shell
 dagger call read-file --source=/my/file/path/README.md
 ```
+</TabItem>
+</Tabs>
 
 And here is an example of passing a file from a remote Git repository as argument:
 
+<Tabs groupId="shell">
+<TabItem value="System shell">
+```shell
+dagger -c 'read-file https://github.com/dagger/dagger.git#main:README.md'
+```
+</TabItem>
+<TabItem value="Dagger Shell">
+```shell title="Type 'dagger' to start Dagger Shell in interactive mode and enter the command below."
+read-file https://github.com/dagger/dagger.git#main:README.md
+```
+</TabItem>
+<TabItem value="Dagger CLI">
 ```shell
 dagger call read-file --source=https://github.com/dagger/dagger.git#main:README.md
 ```
+</TabItem>
+</Tabs>
```

Comment:
Can't comment there but the note bellow has an incorrection. `File` arguments don't support _filters_. That's only for `Directory` arguments.